R/compute_kidney.R

```diff
@@ -137,7 +137,7 @@
 compute_egfr <- function(creat, creatu = "SI", age, weight, sex, race = NULL, method, wt) {
   ### BEGIN DEPRECATION
   if (!missing(wt)) {
-    deprecate_warn("0.12.0", "compute_egfr(old_param = 'wt')", "compute_egfr(new_param = 'weight')")
+    deprecate_stop("0.12.0", "compute_egfr(old_param = 'wt')", "compute_egfr(new_param = 'weight')")
     # old_param is given using exprs()
     weight <- wt
```

Comment:
not a big deal, but this line can never be run. we can delete it because the line above throws an error

Reply:
this applies to a number of other features gone from deprecate warn to stop

Reply:
I think we actually want to return an error to coerce/push the user to use the new argument, as the next step of the deprecation would be to get rid of the old argument all together

Reply:
Ahh yes thats a good point, I'll remove